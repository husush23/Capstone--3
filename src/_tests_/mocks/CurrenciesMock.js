/*eslint-disable*/
import React, {useState} from 'react';
import Navbar from '../../components/Navbar';
import CurrencyMock from './CurrencyMock';

function CurrenciesMock() {
  return (
    <>
      <Navbar />
      <div className='container'>
        <div className='form'>
          <input type='text' />
        </div>
        <div className='cards'>
          <CurrencyMock />
        </div>
      </div>
    </>
  );
}

export default CurrenciesMock;
