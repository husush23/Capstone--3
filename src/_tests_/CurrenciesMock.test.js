/* eslintdisable */
import React from 'react';
import { render } from '@testing-library/react';
import CurrenciesMock from './mocks/CurrenciesMock';

describe('currencies', () => {
  test('Renders currencies correctly', () => {
    render(<CurrenciesMock />);
  });
  test('renders Navbar correctly', () => {
    const { asFragment } = render(<CurrenciesMock />);
    expect(asFragment()).toMatchSnapshot();
  });
});
