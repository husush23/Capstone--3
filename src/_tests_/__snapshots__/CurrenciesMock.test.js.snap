// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`currencies renders Navbar correctly 1`] = `
<DocumentFragment>
  <nav>
    <div />
    <div
      class="p-2"
    >
      <span>
        <i
          class="fa-solid fa-microphone-lines"
        />
      </span>
      <span>
        <i
          class="fa-solid fa-gear"
        />
      </span>
    </div>
  </nav>
  <div
    class="container"
  >
    <div
      class="form"
    >
      <input
        type="text"
      />
    </div>
    <div
      class="cards"
    >
      <div
        class="wrapper"
      >
        <div
          class="card"
        >
          <h3>
            Pair name
          </h3>
          <div
            class="values"
          >
            <div
              class="buy"
            >
              <span>
                ↗
              </span>
                
              <span>
                34.000000
              </span>
            </div>
            <div
              class="sell"
            >
              <span>
                ↘
              </span>
                
              <span>
                35.000000
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
